### PR TITLE
🚀 [deploy] Add deployment scripts

### PR DESCRIPTION
**Pull request by approximator**

### CODE REVIEW

File: .github/scripts/deploy/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+---
+- name: deploy
```
[the remark this answers is not in the document]
Comment (approximator) [reply .github/scripts/deploy/deploy.yml]: Fixed

File: .github/scripts/deploy/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        - /opt/django-chat/db
+        - /opt/django-chat/ui
+        - /etc/traefik
+        - /etc/traefik/certificates
+
+    - name: Recreate chat container
+      become: true
+      vars:
+        container_labels:
+        - key: "traefik.enable"
```
[the remark this answers is not in the document]
Comment (approximator) [reply .github/scripts/deploy/deploy.yml]: Fixed

File: .github/scripts/deploy/files/vault1.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+$ANSIBLE_VAULT;1.1;AES256
```
[the remark this answers is not in the document]
Comment (approximator) [reply .github/scripts/deploy/files/vault1.yml]: Fixed

File: .github/scripts/deploy/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+        name: traefik
+        image: traefik:v2.10
+        state: "started"
+        recreate: true
+
+        volumes:
+          - /var/run/docker.sock:/var/run/docker.sock
+
+        mounts:
+        - type: bind
```
[the remark this answers is not in the document]
Comment (approximator) [reply .github/scripts/deploy/deploy.yml]: Fixed

**Review by oliabn — APPROVED** [review]

**Pull request merged**